### DISCUSSION TITLE
Music ID from an MP3 stream

### DISCUSSION THREAD

**Discussion by Fayrewood** [music-assistant/support#2194] [Feature requests and ideas]
I have Icecast2 running on a Pi to stream from my turntable.  Obviously there's no embedded MP3 tags.  Is there a way to share a snippet of the stream with something like Shazam / Gracenote to get the current album, track etc that can then be displayed in HA?

**Comment by OzGav** [comment]
Not at the moment. On the backlog there is a task to investigate using acousticid to fill in missing metadata. Will be some months away for that though.
